lib/responsive_value.dart

```diff
@@ -189,13 +189,13 @@ class Condition<T> {
       ', condition: ' +
       condition.toString() +
       ', value: ' +
-      value?.toString() +
+      value.toString() +
       ')';
 
   int sort(Condition a, Condition b) {
     if (a.breakpoint == b.breakpoint) return 0;
 
-    return (a.breakpoint < b.breakpoint) ? -1 : 1;
+    return (a.breakpoint! < b.breakpoint!) ? -1 : 1;
```

Comment:
From what I see, nothing prevents `a.breakpoint` or `b.breakpoint` from being null at this point, correct? If so, you'd be comparing null to an int.

Reply:
Thanks for point this out. I double checked and the preceeding `if (e.breakpoint == null)` deliberately throws if the breakpoint is null and a matching named breakpoint is not found. 
The code should be good here.